src/nodes.coffee

```diff
@@ -3097,6 +3114,13 @@ exports.If = class If extends Base
 
 UTILITIES =
   modulo: -> 'function(a, b) { return (+a % (b = +b) + b) % b; }'
+  _boundMethodCheck: -> "
```

Comment:
Is there a reason this helper’s name starts with an underscore? Our other helpers’ names don’t.